encryption/README.md

```diff
@@ -0,0 +1,52 @@
+# Encryption
+
+## What is it?
+
+Encryption involves complex mathematics to obscure data so that it is almost impossible to read without the correct decryption keys or passwords. End-to-end encryption is commonly used when transferring information that needs to be kept secret over a public system such as the internet.
```

Comment:
I always went with this:
https://en.wikipedia.org/wiki/Capitalization_of_%22Internet%22#Regional_differences

>According to Oxford Dictionaries Online, as of April 2016 the spelling Internet remains more usual in the US, while internet has become predominant in the UK.